xbmc/cores/VideoPlayer/VideoPlayer.h

```diff
@@ -347,6 +347,7 @@ class CVideoPlayer : public IPlayer, public CThread, public IVideoPlayer,
   unsigned int GetOrientation() const override;
   void TriggerUpdateResolution() override;
   bool IsRenderingVideo() const override;
+  bool IsLiveStream() const;
   bool Supports(EINTERLACEMETHOD method) const override;
```

Comment:
The new method needs to be added to the `IPlayer` interface, then overridden here.